### PR TITLE
Heppy 8 0 5

### PR DESCRIPTION
**Pull request by cbernet**
Same as #14234 
Let's use the other one for discussion, if you agree.

### CODE REVIEW

**Comment by davidlange6** [comment]
please test

File: EgammaAnalysis/ElectronTools/interface/EnergyScaleCorrection_class.hh [hunk 1]
```diff
@@ -0,0 +1,208 @@
+#ifndef EnergyScaleCorrection_class_hh
```
Comment (davidlange6) [inline EgammaAnalysis/ElectronTools/interface/EnergyScaleCorrection_class.hh]: @cbernet - could you rename this as EnergyScaleCorrection_class.h?
Thanks

Comment (cbernet) [inline EgammaAnalysis/ElectronTools/interface/EnergyScaleCorrection_class.hh]: Hi David, 

Sorry for the delay I had missed this comment. I'm reluctant to do this modification to a file of a package that I am not administrating. I would propose to merge this PR as it is, and to schedule a central check for wrong file names in CMSSW later on.

Comment (cbernet) [inline EgammaAnalysis/ElectronTools/interface/EnergyScaleCorrection_class.hh]: Hi David - Please forget about my previous comment. I had another look and saw that this file is new, so there should be no problem. I have just renamed the file.

**Comment by cbernet** [comment]
Hello, I see that the tests are pending. Could you please restart them?

**Comment by davidlange6** [comment]
please test

**Comment by cbernet** [comment]
Hello, could you please merge this PR? 
Many thanks, 

Colin

**Comment by cbernet** [comment]
Hello, 

Would it be possible to merge this PR? 

I fixed the only problem that has been reported: 
https://github.com/cms-sw/cmssw/pull/14236#discussion_r61054382
https://github.com/cms-sw/cmssw/pull/14236/commits/435e29203f9b1666d9f13432e47b2e151396f83b

Moreover, heppy does not affect any other package in CMSSW. 

If you think these is any issue with this PR, please just let me know so that I can fix it. 
Thank you, 

Colin

**Comment by cbernet** [comment]
@davidlange6 

I am really sorry to insist again, I know these times are hectic. 
But could you please merge this PR or let me know what's wrong with it? 
As far as I can tell, it's been ready for two weeks.

Again, heppy does not affect other packages in CMSSW, 
and we put quite a bit of effort into the porting to CMSSW 80X. 

Cheers & thanks

**Comment by cbernet** [comment]
@davidlange6 thank you very much for merging this PR.

Just to let you know, there is also this backport: 
https://github.com/cms-sw/cmssw/pull/14234
Could you please consider merging it as well? 

Thanks again, 

Colin

**Pull request merged**